src/SilverStripe/InvalidationExtension.php

```diff
@@ -62,6 +62,14 @@ public function onAfterWrite()
     /**
      * @return void
      */
+    public function onAfterReorder()
+    {
+        $this->onChange('write');
```

Comment:
Being really picky, but can the action here be `reorder` instead of `write`? The action parameter exposes extra context in invalidation rules, and reordering is different to writing.